dist/stat_data.py

```diff
@@ -1015,6 +1015,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_prefetch', 'pages requested from the cache due to pre-fetch'),
     CacheStat('cache_pages_requested', 'pages requested from the cache'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_checkpoint', 'pages read into cache by the checkpoint'),
```

Comment:
Nitpick: `by the checkpoint` -> `by checkpoint`

Reply:
Fixed.